### PR TITLE
add beagle-lib-2.1.2-intel-2015a-CUDA-7.5.18.eb

### PR DESCRIPTION
**Pull request by molden**
depends on https://github.com/hpcugent/easybuild-easyconfigs/pull/2039

### CODE REVIEW

**Comment by molden** [comment]
By the way as a reference the added patch resolves this issue: https://github.com/beagle-dev/beagle-lib/issues/82 .

**Comment by boegel** [comment]
@molden: please add a small description (+ ref above) + author at the top of the patch (anything above the `---` line is a comment in a patch)

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/11b09d9022ab4f2672ab for a full test report.

**Comment by boegel** [comment]
@molden: doesn't work for me?

File: easybuild/easyconfigs/b/beagle-lib/beagle-lib-2.1.2-intel-2015a-CUDA-7.5.18.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = 'beagle-lib'
+version = '2.1.2'
+versionsuffix = '-CUDA-7.5.18'
```
Comment (boegel) [inline easybuild/easyconfigs/b/beagle-lib/beagle-lib-2.1.2-intel-2015a-CUDA-7.5.18.eb]: avoid hardcoding the CUDA version in two places:

```python
cudaver = '7.5.18'
versionsuffix = '-CUDA-%s' % cudaver
...
dependencies = [
    ('CUDA', cudaver, '', True),
]

**Comment by molden** [comment]
@boegel will only work when cuda driver is installed.

**Comment by migueldiascosta** [comment]
@molden, this PR is being closed for the following reason(s): no activity for > 1 year.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**